incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,44 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
```

Comment:
Thanks for suggestion. I added `livenessProbe` command to work with community image without additional script. I would prefer to use community image as it would be a good starting point for everyone who has less knowledge to maintain it. But it is also has an option to choose custom image by editing `values.yaml`.

I didn't add `readinessProbe` since it will not be useful for Cassandra. Cassandra do not need Kubernetes to block traffic when it is not ready. Cassandra determine node readiness from read/write consistency level.